app/src/test/java/ru/yandex/praktikumchatapp/ChatViewModelTest.kt

```diff
@@ -32,12 +38,23 @@ class ChatViewModelTest {
     @Test
     fun `send message should update messages with MyMessage`() = runTest {
         val message = Message.MyMessage("TestMessage")
-
+        viewModel.sendMyMessage("TestMessage")
+        assertEquals(listOf(message), viewModel.messages.first())
     }
 
     @Test
     fun testReceiveMessage_concurrentMessages() = runTest {
         val messagesToSend = (1..100).map { Message.MyMessage("Message $it") }
-
+        coroutineScope {
+            val jobs = messagesToSend.map { myMessage ->
+                launch {
+                    viewModel.sendMyMessage(myMessage.text)
+                }
+            }
+            jobs.joinAll()
```

Comment:
Можно не выделять в отдельную переменную, а сразу после `map` вызвать `joinAll()`:
```suggestion
            messagesToSend.map { myMessage ->
                launch {
                    viewModel.sendMyMessage(myMessage.text)
                }
            }.joinAll()
```